### ISSUE TITLE
Highlight different types of json

### ISSUE THREAD

**Issue by usernamehw** [RandomFractals/vscode-data-table#51]
Highlight data with the same colors as it uses in `.json` file in editor, for example `string` | `boolean` | `number`:

![Screenshot (145)](https://user-images.githubusercontent.com/9638156/126835023-40d155c3-3aa3-467e-8649-c62350a93c6f.png)

![Screenshot (146)](https://user-images.githubusercontent.com/9638156/126835190-a8e38b87-c1f1-40f1-9c9c-fecaf9b83012.png)

**Comment by RandomFractals** [comment]
interesting idea. can def. help to differentiate data types in table columns. I like it!

**Comment by RandomFractals** [comment]
@usernamehw btw, if we go this route, should we add separate CSV tab and create rainbow CSV view?

Data analysts and devs seem to like it a lot: https://github.com/mechatroner/vscode_rainbow_csv

**Comment by usernamehw** [comment]
Yes, coloring columns with random colors for `.csv` would be an improvement.

Another feature might be coloring based on column name or column data pattern:

- If the column name includes `date` => then make items of the color that user specified dates should be
- If the table cell value matches RegExp `/\d+/` => then make items of that column of the color user specified numbers should be

I don't know if `.csv` columns are always of the same data type or not though.
